components/collection/CollectionInfo.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <div class="is-flex is-justify-content-space-between flex-direction">
+    <div class="is-flex is-flex-direction-column limit-width">
+      <div v-if="collectionInfo?.currentOwner" class="is-flex mb-2">
+        <div class="mr-2">{{ $i18n.t('activity.creator') }}</div>
+        <IdentityIndex
+          :address="address"
+          hide-identity-popover
+          :show-clipboard="false"
+          class="has-text-link" />
+      </div>
+      <div>
+        {{ collectionInfo?.meta.description }}
+      </div>
+    </div>
+    <div>
+      <div class="columns is-mobile">
+        <div class="column">
+          <CollectionInfoLine
+            :title="`${$i18n.t('activity.network')}`"
```

Comment:
```suggestion
            :title="$t('activity.network')"
```

Reply:
that alone give TS error

:title="`${$t('activity.network')}`" 
to get rid of the error

Reply:
- We will solve this ts error in another issue as it's the most common usage in the codebase.
- There is still some `${$i18n` usage in your PR, pls change them.